README.md

```diff
@@ -119,13 +119,13 @@ Each module is independently versioned using [Lerna](https://github.com/lerna/le
 ## Version Support
 
 At any given time, we support three major versions of Canvas Kit: previous, current, and next. Each
-of these have different levels of support.
+of these has different levels of support.
```

Comment:
```suggestion
of these have different levels of support.
```

Reply:
This is a tricky one that I mess up frequently. I believe @aditya7302 is correct, though. It should be "Each has" not "Each have."

https://www.masterclass.com/articles/is-each-singular-or-plural